docs/data-operations-manual/Reference/Performance-db-tables-construction.md

```diff
@@ -0,0 +1,186 @@
+    - `reporting_historic_endpoints`: To store historical data on endpoints sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+    - `column_field`: This is a mapping table that connects specific columns from an endpoint to their corresponding fields sourced according to the dataset's specification e.g UID -> reference,  sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+
+**Columns**:
+- `organisation`: Extracted from the `provision` table.
+- `organisation_name`: Extracted from the `provision` table.
+- `cohort`: Extracted from the `provision` table.
+- `dataset`: Extracted from the `provision` table.
+- `collection`: Extracted from the `reporting_historic_endpoints` table.
+- `pipeline`: Extracted from the `reporting_historic_endpoints` table.
+- `endpoint`: Extracted from the `reporting_historic_endpoints` table.
+- `endpoint_url`: Extracted from the `reporting_historic_endpoints` table.
+- `resource`: Extracted from the `reporting_historic_endpoints` table.
+- `resource_start_date`: Extracted from the `reporting_historic_endpoints` table.
+- `resource_end_date`: Extracted from the `reporting_historic_endpoints` table.
+- `latest_log_entry_date`: Extracted from the `reporting_historic_endpoints` table.
+- `mapping_field`: Generated using conditional aggregation from `column_field` to identify fields that map correctly to columns.
+- `non_mapping_field`: Generated using conditional aggregation from `column_field` for fields that do not map correctly.
+
+
+
+### 3. Table: [endpoint_dataset_summary](https://datasette.planning.data.gov.uk/performance/endpoint_dataset_summary)
+
+**Purpose**: To summarize endpoint information, including the latest statuses and exceptions.
+
+**Base Tables**:
+    - `endpoint`: Contains information about endpoints sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+    - `source`: Provides a link between endpoints and organizations sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+    - `log`: Contains logs related to endpoint performance sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+
+**Columns**:
+- `endpoint`: Extracted from the `endpoint` table.
+- `endpoint_url`: Extracted from the `endpoint` table.
+- `organisation`: Extracted from the `source` table.
+- `dataset`: Extracted from the resource-dataset mapping.
+- `latest_status`: Extracted from the `log` table.
+- `latest_exception`: Extracted from the `log` table.
+- `entry_date`: Entry date of the endpoint.
+- `end_date`: End date of the endpoint.
+- `latest_resource_start_date`: Start date of the most recent resource.
+
+
+### 4. Table: [provision_summary](https://datasette.planning.data.gov.uk/performance/provision_summary)
+
+**Purpose**: To summarize provision metrics across organizations and datasets, counting various types of issues and endpoints.
```

Comment:
explain what provision metrics means and type of issues might be confusing (user might think it like invalid org etc etc)